flux-plugin/src/flux-runtime/runtime.tsx

```diff
@@ -0,0 +1,112 @@
+/*
+Get all pods and then from them get the controllers for flux
+
+1. helm-controller
+2. kustomize-controller
+3. notification-controller
+4. source-controller
+5. image-reflector-controller
+6. image-automation-controller
+
+*/
+import { K8s } from '@kinvolk/headlamp-plugin/lib';
+import { Link, SectionBox, Table } from '@kinvolk/headlamp-plugin/lib/components/common';
+import { Link as MuiLink } from '@mui/material';
+
+export default function FluxRunTime() {
+  const [pods] = K8s.ResourceClasses.Pod.useList();
+  const [crds] = K8s.ResourceClasses.CustomResourceDefinition.useList();
+  const helmController = pods?.filter(pod => pod.metadata.labels['app'] === 'helm-controller');
```

Comment:
helm-controller is optional, if it doesn't exists this doesn't mean Flux is not installed. The only required controller is source-controller.